CODEOWNERS

```diff
@@ -162,3 +162,6 @@ tests/python_api_testing/conv/ @tt-nshanker
 tests/python_api_testing/unit_testing/fallback_ops @tt-aho
 scripts/profiler/ @mo-tenstorrent
 scripts/install @tapspatel
+scripts/docker @ttmchiou @TT-billteng @tt-rkim
+
+dockerfile @ttmchiou @TT-billteng @tt-rkim
```

Comment:
POSIX new line
<img width="85" alt="Screenshot 2024-05-17 at 10 36 50 AM" src="https://github.com/tenstorrent/tt-metal/assets/109366641/9d6cbcca-e28c-4d66-94c5-c55c0390c000">

You gotta fix your editor man, lol